### PR TITLE
Feature/translations

### PR DESCRIPTION
**Pull request by tuj**

### CODE REVIEW

File: web/modules/custom/genlyd_maps/js/genlyd_maps_tabs.js [hunk 1]
```diff
@@ -19,7 +19,7 @@
         event.stopPropagation();
 
         if (!mapTab.is(':visible')) {
-          btn.text(Drupal.t('Show table'));
+          btn.text(Drupal.t('Show list'));
```
Comment (magnify) [inline web/modules/custom/genlyd_maps/js/genlyd_maps_tabs.js:22]: As long as the javascript does not support Drupal.t, we should avoid using it at all.

For now my suggestion is using the danish words directly. If we, at a later time need to translate it, we should extend it so it works with Drupal.

File: web/themes/custom/genlyd_aarhus/templates/form/form--multistep-form-one.html.twig [hunk 2]
```diff
@@ -13,25 +13,25 @@
...
+    title: ("Hello" | trans) ~ " " ~ user.account.name,
     subtitle: "Let us create an activity" | trans
   },
   chooseHeadline: {
     title: "Choose Headline and Description" | trans
   },
   helpBoxTitle: {
     title: "What is a good headline?" | trans,
-    description: "En god overskrift er mange ting. Den kan være kort og tydelig (fx Vinsmagning i Dokk1). Den kan gøre en nysgerrig (fx Løgn og latin i sognegården) eller bruge ordspil (Luther og lagkage). Den kan love noget (Lær at binde din egen flue) eller være inviterende (Drop-ind tennis)" | trans
+    description: "En god overskrift er mange ting. Den bør være kort og tydelig (fx Vinsmagning i Dokk1). Den kan gøre en nysgerrig (fx Løgn og latin i sognegården) eller bruge ordspil (Luther og lagkage). Den kan love noget (Lær at binde din egen flue) eller være inviterende (Drop-ind tennis)" | trans
```
Comment (magnify) [inline web/themes/custom/genlyd_aarhus/templates/form/form--multistep-form-one.html.twig]: Should this be in english for consistency?

**Review by magnify — CHANGES_REQUESTED** [review]
See comments.

File: web/modules/custom/genlyd_maps/js/genlyd_maps_tabs.js (hunk 1 above)
Comment (tuj) [reply web/modules/custom/genlyd_maps/js/genlyd_maps_tabs.js:22]: These works. It is only the ones in the pop up on the screen that do not work

Comment (tuj) [reply web/modules/custom/genlyd_maps/js/genlyd_maps_tabs.js:22]: Fixed in other feature/activities instead

**Pull request merged**